src/datasources/tag/__snapshots__/TagDataSource.test.ts.snap

```diff
@@ -430,12 +430,14 @@ exports[`queries numeric tag history 1`] = `
     "fields": [
       {
         "name": "time",
+        "type": "time",
         "values": [
-          1672531200000,
-          1672531260000,
```

Comment:
This shouldn't be changing. In my experience, some Grafana visualizations don't handle string timestamps correctly. We should be converting the timestamps to unix milliseconds.

Reply:
@mure We reviewed your suggestions and understand your perspective, but we also have this working in SLS as Tigran currently has the runQuery returning what the visualization requires.  Given our time constraints, is this necessary to move forward or can we move forward as-is?

Reply:
@mure If we've tested this in SLS and it satisfies the requirements for AAU, then I won't block the review on this.

Reply:
@TigranVardanyan @markacamps 

If the correct thing to do is convert these timestamps to unix milliseconds, that is what we should do. It shouldn't be a large amount of work, either. We need to remember that SLS isn't the only client of these plugins and just because it works there doesn't mean it does everywhere or in every edge-case scenario.

Reply:
@cameronwaterman I will fix milliseconds part. What about gaps in the chart, it fixes by **Connect null values** option
Earlier I tried many times to return separated data frame for each timeframe, but with no success, only way I founded to mix frames to one and fill it with null values

Reply:
I still think multiple data frames is the better approach, but with that workaround I'm okay with submitting this for now

Reply:
@cameronwaterman  @mure done